pkg/kubecli/kubecli.go

```diff
@@ -53,40 +54,37 @@ func GetFromFlags(master string, kubeconfig string) (*kubernetes.Clientset, erro
 	config.APIPath = "/apis"
 	config.ContentType = runtime.ContentTypeJSON
 
-	return kubernetes.NewForConfig(config)
+	return rest.RESTClientFor(config)
 }
 
-func Get() (*kubernetes.Clientset, error) {
-	return GetFromFlags(master, kubeconfig)
-}
+func GetCoreFromFlags(master string, kubeconfig string) (*kubernetes.Clientset, error) {
+	config, err := clientcmd.BuildConfigFromFlags(master, kubeconfig)
+	if err != nil {
+		return nil, err
+	}
+
+	config.GroupVersion = &v1.GroupVersion
```

Comment:
I think that these four lines do have no effect for the k8s core client. They only have an effect when we configure our kubevirt rest client. I think we can remove them.